README.md

```diff
@@ -16,3 +16,60 @@ $> cp sample.env .env
 ```
 $> npm start
 ```
+
+## Localization
+
+In this project we're using [React-Intl](https://github.com/yahoo/react-intl) to localize our application and YAML for translation.
+
+#### Localize a component or page
+
+To localize a component or page you have to always include IntlMixin in your class mixins for example:
+
+``` typescript
+var React = require('react');
+var IntlMixin = require('react-intl').IntlMixin;
+
+var Example = React.createClass({
+	mixins: [IntlMixin]
+  render: function() {
+    return (
+      <div>
+      	<h1>{this.getIntlMessage('key_name_here')}
+      </div>
+    );
+  }
+
+});
+```
+
+Once you added the mixin it will expose `getIntlMessage` method for your component to get the localized message for the given key name.
+
+#### Adding locale
+
+Since we are using YAML for our translation, but React-Intl expect JSON which will require an extra build step to convert from YAML to JSON.
```

Comment:
Maybe better?

"Because we are using YAML for our translation and React-Intl expects JSON, we need an extra build step to convert YAML to JSON"